nnstreamer_example/custom_example_opencv/nnstreamer_customfilter_opencv_scaler.cc

```diff
@@ -171,6 +171,16 @@ pt_allocate_invoke (void *private_data,
   return 0;
 }
 
+/**
+ * @brief destroy notify callback of tensor_filter custom
+ */
+static void
+pt_destroy_notify (void * data)
+{
+  assert (data);
```

Comment:
assert() , is it necessary?
If it is necessary for the test, use g_assert().

Reply:
it was changed :)